legion/engine/physics/systems/physicssystem.hpp

```diff
@@ -129,9 +121,12 @@ namespace legion::physics
     private:
 
         static std::unique_ptr<BroadPhaseCollisionAlgorithm> m_broadPhase;
-        const float m_timeStep = 0.02f;
 
+        const float m_timeStep = 0.02f;
+        float m_accumulator = 0.0f;
 
+        size_t m_maxTimeStepPerFrame = 3;
```

Comment:
`size_t` is part of the `std::` namespace, so this would either require that namespace attached, or `size_type` should be used. `size_type` is the rythe default and technically more correct in this case.